src/types.ts

```diff
@@ -2,3 +2,59 @@ export type AppError = {
   message: string
   details?: unknown
 }
+
+export type User = {
+  id: string
+  firstName: string
+}
+
+export type Item = {
+  id: string
+  amount: number
+}
+
+export type Dish = {
+  id: string
+  name: string
+  recipe?: string
+  source?: string
+  last?: string
+  items: Item[]
+}
+
+export type Proposal = {
+  date: string
+  dishId: string
+  served: boolean
+}
+
+export type Ingredient = {
+  id: string
+  name: string
+  unit: string
+  group: string
+}
+
+export type CompleteItem = {
+  id: string
+  name: string
+  unit: string
+  group: {
+    id: string
+    title: string
+    order: number
+  }
+  amount: number
+  originalAmount?: number
+}
+
+export type Unit = {
+  id: string
+  name: string
+  increment: number
+}
+
+export type ItemGroup = {
+  order: number
+  title: string
+}
```

Comment:
How certain are you, that these types are going to stay the way they are? The alternative would be to pull them in from the old code once we need them.

Reply:
You're right, I should do that. Thanks for approval!